### ISSUE TITLE
Logo for `graphblas-algorithms`

### ISSUE THREAD

**Issue by eriknw** [python-graphblas/graphblas-algorithms#76]
Now that we have a logo for `python-graphblas` (https://github.com/python-graphblas/python-graphblas/pull/506), I suppose the logical next step is to make a logo for `graphblas-algorithms`. It would probably make sense to use a variant of the `python-graphblas` logo.

Here's one idea:
![ga_path](https://github.com/python-graphblas/graphblas-algorithms/assets/2058401/424b1e9c-ec77-4198-af1f-1fa6a48663e7)

**Comment by SultanOrazbayev** [comment]
Looks cool, but the styles for g and a are rather distinct IMO. (not to say I have better ideas)

**Comment by eriknw** [comment]
I agree, styles are very weird, and the `@` doesn't screem "matrix multiply" w/o something to the right of it. Here's a G and A that are a bit more similar:
![ga2](https://github.com/python-graphblas/graphblas-algorithms/assets/2058401/fa831b82-abdb-4eb2-bc87-9f648211bf09)

I'm all ears for ideas.

btw, the angle of the "G" is 12.458910191690801 degrees.

**Comment by eriknw** [comment]
Another:
![ga3](https://github.com/python-graphblas/graphblas-algorithms/assets/2058401/02775391-ab6c-4be2-a99d-ba7ab47825f0)
and the python-graphblas logo shown for reference:
![python-graphblas-logo](https://github.com/python-graphblas/graphblas-algorithms/assets/2058401/6e6aec00-64e1-4d6d-8af2-f007cc8eaa98)
